blocks/inline-blocks/inline-image.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default {
```

Comment:
I like how this is similar to normal block registration! :) Maybe also `export const settings`?

Reply:
Great idea! I used your suggestion and export `name` and `settings` now, similar to standard blocks https://github.com/WordPress/gutenberg/pull/5794/commits/4403d200433ded292223d7c9bb849b285eabbce5.